### ISSUE TITLE
Add libxrpl version to server_info

### ISSUE THREAD

**Issue by godexsoft** [XRPLF/clio#853]
It would be useful to know what version of libxrpl clio is built against. For example to know what amendments are supported etc. Let's expose it thru `server_info`.
See https://github.com/ximinez/rippled/blob/develop/src/ripple/protocol/impl/BuildInfo.cpp#L36

**Comment by intelliot** [comment]
This is fine, although I think it would be preferable to bump the version of Clio each time libxrpl is updated. That way, I can know what I'm running just by looking at one version number. For example, maybe it's something like (made up numbers):
- Clio 1.1.0 is built with libxrpl 1.11.0
- Clio 1.2.0 is built with libxrpl 1.12.0
- Clio 1.3.0 is built with libxrpl 2.0.0
- Clio 2.0.0 is built with libxrpl 2.0.0
- (or whatever the case may be)

In other words, a one-to-many mapping would make it possible for me to look at one number (Clio version) and understand what is supported, and whether I need to upgrade.

**Comment by godexsoft** [comment]
We should document the `libxrpl` version used as part of release notes. I still like to have this in `server_info` too just so we can always clearly see what it is without having to reach for release notes.
